protopipe_grid_interface/scripts/submit_jobs.py

```diff
@@ -137,6 +139,11 @@ def split_input_data(dirac, filelist, n_file_per_job):
 else:
     switches["tag"] = str(switches["tag"])
 
+if "particle" in switches:
+    if switches["particle"] not in ["gamma", "electron", "proton"]:
+        print("Invalid option for --particle: ", switches["particle"])
+        sys.exit(1)
```

Comment:
I think we should add here a check that `--particle` is **always** initialized to 1 of the supported choices (like `argparse`'s `choices` option).

Also you can do directly this,

`sys.exit('ERROR: --particle can be only 1 of ["gamma", "electron", "proton"]')`

Reply:
>  think we should add here a check that --particle is always initialized to 1 of the supported choices (like argparse's choices option

Isn't that what it does? It fails if it is not one of the three possiblities

> Also you can do directly this,

I thought `sys.exit` takes an exit status integer, but you're right - if you give it anything other than the statius integer you want, it returns 1 (general failure) and prints the status. I'll use that

Reply:
> Isn't that what it does? It fails if it is not one of the three possiblities

yes and no: right now if `particle` is not in `grid.yaml` and not specified via the CLI nothing happens and it will crash